### ISSUE TITLE
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.ImageView.setImageResource(int)' on a null object reference

### ISSUE THREAD

**Issue by Kwazir** [idnow/de.idnow.android.sdk#38]
Hi,

I'm using 4.6.0 SDK and after process of capturing personal ID in the camera, I'm getting fatal error:

```
2020-12-10 17:22:02.648 29778-29778/{package} E/AndroidRuntime: FATAL EXCEPTION: main
    Process: sportbet.android.vikings.debug, PID: 29778
    java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.ImageView.setImageResource(int)' on a null object reference
        at de.idnow.core.ui.main.k0.onBindViewHolder(IDnowListCountryAdapter.java:10)
        at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7065)
        at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7107)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6012)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6279)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6118)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6114)
        at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2303)
        at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1627)
        at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1587)
        at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:665)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4134)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3851)
        at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4404)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at androidx.viewpager.widget.ViewPager.onLayout(ViewPager.java:1775)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at androidx.constraintlayout.widget.ConstraintLayout.onLayout(ConstraintLayout.java:1915)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1081)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at com.android.internal.policy.DecorView.onLayout(DecorView.java:785)
        at android.view.View.layout(View.java:19655)
        at android.view.ViewGroup.layout(ViewGroup.java:6127)
        at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2600)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2309)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1467)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7306)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:979)
        at android.view.Choreographer.doCallbacks(Choreographer.java:791)
2020-12-10 17:22:02.648 29778-29778/{package} E/AndroidRuntime:     at android.view.Choreographer.doFrame(Choreographer.java:726)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:965)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6759)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:770)
```

**Comment by linhnguyen-idnow** [comment]
Thanks @Kwazir for your report. Can you give us the ident-id?
We will check it on our side and answer you soon.

**Comment by Kwazir** [comment]
TST-VLWGW

**Comment by Kwazir** [comment]
> Thanks @Kwazir for your report. Can you give us the ident-id?
> We will check it on our side and reply you soon.

TST-VLWGW

**Comment by linhnguyen-idnow** [comment]
@Kwazir could you please clean project and build again?

**Comment by Kwazir** [comment]
> @Kwazir could you please clean project and build again?
I proceeded with:
- Invalidate Caches / Restart
- removed cache with libraries and downloaded it again

The result was the same.

**Comment by Kwazir** [comment]
@linhnguyen-idnow I did many tests and below are my conclusions.
The reason of receiving fatal error, is connected with using another SDK from idNow:
https://github.com/idnow/de.idnow.android
at the same time.

I need both of them in my single application and IMHO it should be possible, because they're comming form the same company.

Can you help me with that?

**Comment by linhnguyen-idnow** [comment]
@Kwazir Thanks for your information, It's really help. I will check and back to you soon.

**Comment by Kwazir** [comment]
> @Kwazir Thanks for your information, It's really help. I will check and back to you soon.

@linhnguyen-idnow , when this issue will be fixed?

**Comment by linhnguyen-idnow** [comment]
@Kwazir I tried to include 2 sdk in one hostApp, but It worked well on my side. 
not sure how can i help you in this case. 
@SlawomirLeski do you have any idea?

**Comment by linhnguyen-idnow** [comment]
@Kwazir Can you give me the video to reproduce your issue?

**Comment by Kwazir** [comment]
@linhnguyen-idnow I recordeded it, but it's too big to upload :(

**Comment by Kwazir** [comment]
@linhnguyen-idnow 
https://user-images.githubusercontent.com/21369693/103219715-cbaea900-491e-11eb-8331-98e419ab3d83.mp4

**Comment by Kwazir** [comment]
@linhnguyen-idnow hi the New Year. What about this issue?

> @Kwazir I tried to include 2 sdk in one hostApp, but It worked well on my side.

Maybe you'll be able to send me the project you wrote about? I'll try to compile it on my side.

**Comment by Kwazir** [comment]
@linhnguyen-idnow @SlawomirLeski
IMHO it's connected with the ID I'm using to test it.
SDK can not identify country and trying to ask user, showing him list of country flags.
`IDnowListCountryAdapter` class is using image resources which can not be reached.

**Comment by SlawomirLeski** [comment]
Hallo @Kwazir,

could you share with us token you have used?

Kind regards,
Slawek

**Comment by Kwazir** [comment]
Hi @SlawomirLeski 
all time the same: TST-VLWGW

**Comment by Kwazir** [comment]
@linhnguyen-idnow @SlawomirLeski 
Something has changed in the matter of this issue?

**Comment by linhnguyen-idnow** [comment]
@Kwazir It should be good if you can share to me the sample project that you show in the video. Then I can debug directly on that then I can see what happen with your issue. Could you please?

**Comment by Kwazir** [comment]
> @Kwazir It should be good if you can share to me the sample project that you show in the video. Then I can debug directly on that then I can see what happen with your issue. Could you please?

@linhnguyen-idnow sure, I added you to the sample project you saw in the video.

**Comment by Kwazir** [comment]
@linhnguyen-idnow when can I expect this issue to be resolved?

**Comment by linhnguyen-idnow** [comment]
@Kwazir It will be available in next release

**Comment by linhnguyen-idnow** [comment]
https://user-images.githubusercontent.com/34857505/106460296-b3a1ea80-64c5-11eb-81a4-44c73a6c41cd.mov

But I will inform when it's available.

**Comment by Kwazir** [comment]
> @Kwazir It will be available in next release

summarizing - you confirm this issue exists and next release will fix it, right?

**Comment by linhnguyen-idnow** [comment]
Exactly! @Kwazir . Thanks for your patience.

**Comment by linhnguyen-idnow** [comment]
@Kwazir I can check on  4.7.0 version

**Comment by Kwazir** [comment]
> @Kwazir You can check on 4.7.0 version

The problem has been solved with autoident 4.7.0 / videoident 4.5.1
